### PR TITLE
Content change

### PR DESCRIPTION
**Pull request by kumuthac23**

### CODE REVIEW

File: src/footer/Footer.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ function Footer() {
                   <Fade bottom duration={1100} delay={100}>
                     <Typography variant="h6">Email</Typography>
                     <Typography variant="body1" mb={2}>
-                      Leaders@school.com
+                      theleaderskids@gmail.com
```
Comment (praveensumathi) [inline src/footer/Footer.tsx:113]: Kids or Kidz ?

File: src/footer/Footer.tsx [hunk 2]
```diff
@@ -95,7 +95,7 @@ function Footer() {
                   <Fade bottom duration={1100}>
                     <Typography variant="h6">Phone</Typography>
                     <Typography variant="body1" mb={2}>
-                      123-456-7890
+                      98434 18723, &nbsp;99944 98723
```
Comment (praveensumathi) [inline src/footer/Footer.tsx:98]: can we remove the space between phone numbers

File: src/pages/contact/Contact.tsx [hunk 3]
```diff
@@ -22,17 +22,17 @@ function Contact() {
...
       icon: <LocalPhoneIcon />,
       title: "OUR CONTACT NUMBER",
-      description: "+(91) 7207012005",
+      description: "+(91) 98434 18723",
     },
     {
       icon: <MailOutlineIcon />,
       title: "OUR CONTACT E-MAIL",
-      description: "sunflowerschoolmail@gmail.com",
+      description: "theleaderskids@gmail.com",
```
Comment (praveensumathi) [inline src/pages/contact/Contact.tsx:35]: kids or kidz?

File: src/primarynavbar/Primarynavbar.tsx [hunk 4]
```diff
@@ -52,13 +52,13 @@ function Primarynavbar() {
                 <Box sx={{ ...commonGridItemStyles, gap: 1 }}>
                   <MailOutlineIcon />
                   <Typography sx={commonTypographyStyles}>
-                    sunflowerschoolmail@gmail.com
+                    theleaderskids@gmail.com
```
Comment (praveensumathi) [inline src/primarynavbar/Primarynavbar.tsx:55]: check the mail spelling kids or kidz

File: src/footer/Footer.tsx (hunk 1 above)
Comment (kumuthac23) [reply src/footer/Footer.tsx:113]: kids only..i checked

File: src/footer/Footer.tsx (hunk 2 above)
Comment (kumuthac23) [reply src/footer/Footer.tsx:98]: i removed

File: src/pages/contact/Contact.tsx (hunk 3 above)
Comment (kumuthac23) [reply src/pages/contact/Contact.tsx:35]: kids only

File: src/primarynavbar/Primarynavbar.tsx (hunk 4 above)
Comment (kumuthac23) [reply src/primarynavbar/Primarynavbar.tsx:55]: kids only..i checked

**Review by praveensumathi — APPROVED** [review]

**Pull request merged**